app/src/main/res/values/dimens.xml

```diff
@@ -19,6 +19,7 @@
 
     <dimen name="bar_height">40dp</dimen>
     <dimen name="large_bar_height">58dp</dimen>
+    <dimen name="corner_radius">29dp</dimen> <!--half of large_bar_height-->
```

Comment:
Potential caveat: when not using the large search bar, list view will be more rounded than the favorite bar that will be more rounded than the search bar